evmbin/src/display/std_json.rs

```diff
@@ -113,26 +100,28 @@ impl<Trace: Writer, Out: Writer> vm::Informant for Informant<Trace, Out> {
 
 		match result {
 			Ok(success) => {
-				writeln!(
-					&mut trace_sink,
-					"{{\"stateRoot\":\"{:?}\"}}", success.state_root
-				).expect("The sink must be writeable.");
-				writeln!(
-					&mut out_sink,
-					"{{\"output\":\"0x{output}\",\"gasUsed\":\"0x{gas:x}\",\"time\":{time}}}",
-					output = success.output.to_hex(),
-					gas = success.gas_used,
-					time = display::as_micros(&success.time),
-				).expect("The sink must be writeable.");
+				let trace_data = json!({"stateRoot": success.state_root});
+				writeln!(&mut trace_sink, "{}", trace_data.to_string())
+					.expect("The sink must be writeable.");
+
+				let out_data = json!({
+					"output": format!("0x{}", success.output.to_hex()),
+					"gasUsed": format!("0x{:x}", success.gas_used),
+					"time": display::as_micros(&success.time),
+				});
+
+				writeln!(&mut out_sink, "{}", out_data.to_string())
+					.expect("The sink must be writeable.");
 			},
 			Err(failure) => {
-				writeln!(
-					&mut out_sink,
-					"{{\"error\":\"{error}\",\"gasUsed\":\"0x{gas:x}\",\"time\":{time}}}",
-					error = display::escape_newlines(&failure.error),
-					gas = failure.gas_used,
-					time = display::as_micros(&failure.time),
-				).expect("The sink must be writeable.");
+				let out_data = json!({
+					"error": display::escape_newlines(&failure.error),
```

Comment:
`escape_newlines` is not necessary any more with `serde_json`, please remove that function.